akka-stream-tests/src/test/scala/akka/stream/DslConsistencySpec.scala

```diff
@@ -33,11 +33,11 @@ class DslConsistencySpec extends WordSpec with Matchers {
   val jRunnableGraphClass: Class[_] = classOf[akka.stream.javadsl.RunnableGraph[_]]
   val sRunnableGraphClass: Class[_] = classOf[akka.stream.scaladsl.RunnableGraph[_]]
 
-  val ignore =
+  val ignore: Set[String] =
     Set("equals", "hashCode", "notify", "notifyAll", "wait", "toString", "getClass") ++
       Set("productArity", "canEqual", "productPrefix", "copy", "productIterator", "productElement") ++
       Set("create", "apply", "ops", "appendJava", "andThen", "andThenMat", "isIdentity", "withAttributes", "transformMaterializing") ++
-      Set("asScala", "asJava", "deprecatedAndThen", "deprecatedAndThenMat")
+      Set("asScala", "asJava", "deprecatedAndThen", "deprecatedAndThenMat", "collectType")
```

Comment:
ignored it in the specific cases, it still needs to be consistent across javadsl types